app/views/home/index.html.erb

```diff
@@ -67,6 +67,7 @@
                              autofocus: true,
                              minlength: 11, maxlength: 14,
                              required: true,
+                             type: :cpf,
```

Comment:
Este é um non-conformant HTML5 type, então mesmo que alguns browsers façam fallback para type=text, segundo a [HTML live spec (May 2021)](https://html.spec.whatwg.org/#states-of-the-type-attribute) o atributo não é suportado (esta [versão no GitHub é mais acessível](https://w3c.github.io/html-reference/input.htm)).

Mudando isso para ser algo como um `data: { sel: 'cpf' }` (para ser acessível através de um DOM `[data-sel=cpf]` por exemplo, ou uma CSS class, já resolveria o problema.

Reply:
Obrigado pela revisão, @jmonteiro.
Concordo com as mudanças pedidas.